src/lib/viewers/BaseViewer.js

```diff
@@ -1055,10 +1055,13 @@ class BaseViewer extends EventEmitter {
      * Handler for annotation toolbar region comment button click event.
      *
      * @private
+     * @param {boolean} activeControl - avtive control mode in AnnotationControls
```

Comment:
`acvtive` -> `active`.

I believe we typically represent destructured arguments as:
`@param {boolean} options.activeControl - ...`

Reply:
Thank you for your feedback, I'll pay attention to those in my future PRs. I just removed this line in my latest push.